tardis_env27.yml

```diff
@@ -16,7 +16,7 @@ dependencies:
 - astropy=1.3
 - numexpr=2.6
 - Cython=0.21
-- networkx=1.10
```

Comment:
@lukeshingles can you just leave the version - I think your fix should work with both

Reply:
Done. (Sorry, I think I broke your code review comment when I rebased the branch)